repose-aggregator/core/repose-core-api/src/main/java/org/openrepose/core/services/reporting/metrics/SummingMeterFactory.java

```diff
@@ -19,22 +19,20 @@
  */
 package org.openrepose.core.services.reporting.metrics;
 
-import com.yammer.metrics.core.TimerContext;
-
-import java.util.concurrent.TimeUnit;
+import com.codahale.metrics.Meter;
 
 /**
- * Interface to allow different TimerByCategory implementations to be used interchangeably.
- * <p/>
- * Any class which implements this interface is expected to be thread-safe.  The individual yammer Timer class is
- * thread-safe.
- * <p/>
- * These objects should be created by the {@link org.openrepose.core.services.reporting.metrics.impl.MetricsServiceImpl}
- * factory class.
+ * A factory used to create {@link Meter}s. The {@link Meter}s created by this
+ * factory will have their values tracked and summed. The summed value will be
+ * reported as its own {@link Meter}.
+ * <p>
+ * This factory is hierarchical. Child factory names will be prefixed by the name
+ * of their parent factory. The overall structure of all factories is a tree.
  */
-public interface TimerByCategory {
+public interface SummingMeterFactory {
```

Comment:
I'll go ahead and change the names of things to open the way for other implementations, at least. Leaving the combination of these things in the future sounds good, though.